library/wazo_wizard.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+
+# Copyright: (c) 2019, Wazo Communication Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+
+from wazo_confd_client import Client as Confd
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: wazo_wizard
+
+short_description: initial configuration of a Wazo engine
+
+version_added: "2.4"
+
+description:
+    - "This is my longer description explaining my sample module"
+
+options:
+    hostname:
+        description:
+            - This is the hostname of the wazo engine host
+        required: false
+        default: localhost
+    language:
+        description:
+            - language to use (ex. en_US).
+        required: true
+    password:
+        description:
+            - password for the Wazo engine
+        required: true
+    engine_api_port:
+        description:
+            - password for the Wazo engine
+        required: false
+        default: 443
+    https:
+        description:
+            - use https to connect to the Wazo engine
+        required: false
+        default:true
```

Comment:
```suggestion
        default: true
```
`ansible-doc` does not accept `default:true`